Updates instructions on running mkdocs locally in docker (#4849)
knative/docs: hack/docker/README.md

# Development notes for working with mkdocs

> For details of how to use a container for local development, see the "Use the Docker container" section of the [Previewing Docs Locally](../../contribute-to-docs/getting-started/previewing-docs-locally.md#option-1-use-the-docker-container) page.

Reviewer: ```suggestion
For details about how to use a container for local development, see [Use the Docker container](../../contribute-to-docs/getting-started/previewing-docs-locally.md#option-1-use-the-docker-container) on the Previewing Docs Locally page.
```
